vg/vg.go

```diff
@@ -73,7 +74,10 @@ type Canvas interface {
 
 	// FillString fills in text at the specified
 	// location using the given font.
-	FillString(f Font, x, y Length, text string)
+	FillString(f Font, pt Point, text string)
+
+	// DrawImage draws the image, scaled to fit the destination rectangle.
```

Comment:
Wrap this line a little earlier to be consistent with the other methods - remember that godoc doesn't flow text in a decl block.